src/ui/ButtonNavigator.js

```diff
@@ -21,19 +23,19 @@ module.exports = ButtonNavigator
 ButtonNavigator.prototype.render = function () {
   var self = this
   var view = yo`<div>    
-    <button id='overback' onclick=${function () { self.trigger('stepOverBack') }} disabled=${this.overBackDisabled} >
+    <button id='overback' style=${ui.formatCss(style.button)} onclick=${function () { self.trigger('stepOverBack') }} disabled=${this.overBackDisabled} >
```

Comment:
That is how React manage styling, using the style property and not class.
https://facebook.github.io/react/tips/inline-styles.html
I did not changed this behavior when we moved to yoyo. We could switch to use class if it's necessary, but in another pr please :)